amp-labs/react
# [ENG-1549] feat: add error state to installation update read

### Summary
adds error handling to the update read installation flow. This PR is followup to the error handling for create read https://github.com/amp-labs/react/pull/543

#### does not include
- update installation write
- update installation read

#### demo
![demo-update-installation-error](https://github.com/user-attachments/assets/329d3313-66b1-447a-a4f4-e38c1f38ff44)

#### testing steps
1. create an installation with object that is set up properly
2. switch to an object that does not have a destination and attempt to save.
3. installation update should fail.
